src/configuration/SSH/OpenSSH/7.3/sshd_config

```diff
@@ -0,0 +1,118 @@
+#-----------------------------
+# Daemon
+#-----------------------------
+
+# Specifies the port number that sshd(8) listens on.
+# The default is 22.
+#Port 1022
+
+# Specifies the local addresses sshd(8) should listen on.
+# The default is to listen on all local addresses.
+#ListenAddress 192.168.23.42
+
+# Specifies a file containing a private host key used by SSH.
+# The default is
+## /etc/ssh/ssh_host_dsa_key,
+## /etc/ssh/ssh_host_ecdsa_key,
+## /etc/ssh/ssh_host_ed25519_key and
+## /etc/ssh/ssh_host_rsa_key for protocol version 2.
+HostKey /etc/ssh/ssh_host_ed25519_key
+HostKey /etc/ssh/ssh_host_rsa_key
+
+# Specifies the maximum number of concurrent unauthenticated connections
+# to the SSH daemon.
+# The default is 10:30:100.
+MaxStartups 3
+
+# Specifies the maximum number of authentication attempts permitted per
+# connection.
+# The default is 6.
+MaxAuthTries 1
+
+# Specifies the maximum number of open shell, login or subsystem (e.g. sftp)
+# sessions permitted per network connection.
+# The default is 10.
+MaxSessions 6
+
+#-----------------------------
+# Authentication
+#-----------------------------
+
+# The server disconnects after this time if the user has not
+# successfully logged in.
+# The default is 120 seconds.
+LoginGraceTime 60
+
+# Specifies whether root can log in using ssh(1).
+# The default is "prohibit-password".
+PermitRootLogin no
+
+# Specifies the file that contains the public keys that can be used for
+# user authentication.
+# The default is ".ssh/authorized_keys .ssh/authorized_keys2".
+AuthorizedKeysFile .ssh/authorized_keys
+
+# Specifies whether password authentication is allowed.
+# The default is "yes".
+PasswordAuthentication no
+
+# Specifies whether challenge-response authentication is allowed (e.g.
+# via PAM) or through authentication styles supported in login.conf(5)).
+# The default is "yes".
+ChallengeResponseAuthentication no
+
+# If specified, login is allowed only for user names that match one of
+# the patterns.
+# By default, login is allowed for all users.
+#AllowUsers ed
+
+#-----------------------------
+# Cryptography
+#-----------------------------
+
+# Specifies the available KEX (Key Exchange) algorithms.
+# The default is:
+## curve25519-sha256@libssh.org,
+## ecdh-sha2-nistp256,
+## ecdh-sha2-nistp384,
+## ecdh-sha2-nistp521,
+## diffie-hellman-group-exchange-sha256,
+## diffie-hellman-group14-sha1.
+KexAlgorithms curve25519-sha256@libssh.org,ecdh-sha2-nistp521,diffie-hellman-group-exchange-sha256
+
+# Specifies the ciphers allowed.
+# The default is:
+## chacha20-poly1305@openssh.com,
+## aes128-ctr,
+## aes192-ctr,
+## aes256-ctr,
+## aes128-gcm@openssh.com,
+## aes256-gcm@openssh.com.
+Ciphers chacha20-poly1305@openssh.com,aes256-ctr,aes256-gcm@openssh.com
```

Comment:
We had this discussion on the mailing-list when we started out. We've opted to accept 128bit ciphers in our guide. The ciphersuite we recommend for TLS is set-up similar. I really don't see a reason to exclude aes128 from the OpenSSH deamons available ciphers.

We even wrote with the AES designers about the issue and got the following response (also quoted in the bettercrypto guide):
```
“On the choice between AES256 and AES128: I would never consider using AES256, just like I don’t wear a helmet when I sit inside my car. It’s too much bother for the epsilon improvement in security.”
```